common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -263,7 +266,7 @@ def has_support(self, view, functionality):
         Returns whether the given view has support for the given functionality.
         """
         if functionality == "multi_device":
-            return all(
+            return self.problem_types and all(
```

Comment:
Though I'm not sure, I suspect that the latter is more correct, but I'm curious why you're touching it in this PR.  Is it related to the error we're seeing?